### DISCUSSION TITLE
Oct 14 2023 ecliipse

### DISCUSSION THREAD

**Discussion by ghost** [XEphem/XEphem#82] [Show and tell]
Eclipses are good checks for ephemerides. See attached screen shot comparing the Oct 14 2023 eclipse with the NASA prediction for Albuquerque NM. XEphem is still doing a pretty good job.
<img width="2560" alt="XEphem 2023 Eclipse" src="https://github.com/XEphem/XEphem/assets/12057700/47ccbb86-96fd-4ce0-a235-2ac777ba4fdd">

**Comment by brandon-rhodes** [comment]
Thanks for the example, the results look very close indeed! XEphem is indeed still doing well.

I wasn't, alas, able to make it to the Four Corners area in time to see the annular eclipse — I was picking up two friends at the Phoenix airport that Saturday morning for a Grand Canyon hike — but I did bring eclipse glasses, and was able to see the Sun about 85% covered. It was fun to loan the glasses to one or two passers-by who didn't know an eclipse was happening; one said that he thought there might be something wrong with his eyes as he felt the parking lot was darker and darker.
